doc/api/fs.md

```diff
@@ -4016,6 +4016,7 @@ to be notified of filesystem changes.
 * On SunOS systems (including Solaris and SmartOS), this uses [`event ports`][].
 * On Windows systems, this feature depends on [`ReadDirectoryChangesW`][].
 * On Aix systems, this feature depends on [`AHAFS`][], which must be enabled.
+* On IBM i systems, this feature is not supported.
```

Comment:
I didn't add any line suggesting to use `fs.watchFile()` as this option is mentioned later.